src/components/ActionBar.tsx

```diff
@@ -5,7 +5,7 @@ export type ToolbarVariant = "full-screen" | "primary";
 
 const styles = {
   primary: { bgColor: "green.500", h: "64px", minH: "64px" },
-  "full-screen": { bgColor: "brand.500", h: "4rem" },
+  "full-screen": { bgColor: "gray.500", h: "4rem" },
```

Comment:
The variant prop isn't used in the repo, so we can just drop styles and its use in the component below.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/290/commits/ae025270f9a0b24f7d5fdc059e299df530c9fc16